examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Suggestions:</p><p> </p><ul><li>Simplify the title to "Topological features for time series forecasting"? I personally do not like having the class names in the title, but that's just me being a pedant :) </li><li>"This notebook shows how <code>giotto-tda</code> can be used to create topological features for time series forecasting tasks, and how to ..."</li><li>"a single time series array <code>X</code> of shape <code>(n_timestamps, n_features)</code>  is turned into ..."</li><li>"There are two main issues that arise when building forecasting models with sliding windows:"</li><li>"<code>giotto-tda</code>  provides a selection of transformers ..." (delete "relevant")</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Good suggestions. All implemented.